guide/Guide/5.comment.md

```diff
@@ -42,8 +41,8 @@ Now it's time to check if the comment worked fine
 As you see the comment at line 1 didn't appear in the result
 
 ## Multiple Lines Comment
-In some cases you may want to comment out more then one line, or just a specific part of it.
-You can do that by wrapping the comment with a combination of slashes and asterisks `/* Comment */`
+In some cases you may want to comment out more than one line.
```

Comment:
"specific part of it" meaning got removed. can you add it back in your own way.
by specific part, it means commenting a section of code that exists in one line and not commenting the whole line, like:
"Hello $username?/*, How*/ are you?"

to comment out `, How`